### ISSUE TITLE
Tonight's Game: Princess Tomato and the Salad Kingdom [NES] 

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#21]
Weird first pick, I realize. I was surprised too, but it's been a game I've wanted to really play for a long time and it just jumped out at my browsing through the boxes and carts.

## What do you know about the game?
Not much really.

I bought it as a gift to my better half on eBay because she's vegetarian and thought it would be amusing.

It was an elaborate dad joke, but I have no regrets.

Outside of the weirdness that are sentient vegetables, I know it's a classic PC game port and text adventure. Gameplay seems light, but I always had a thing for these kinds of text adventures on the NES (I.E. Déjà Vu, from Kemco) and want to see what, if anything, makes this one special.

If I remember correctly, there is a battle system in the game, but it's very simple. As many older Japanese titles use, this game uses a Roshambo or Rock, Paper, Scissors sort of random battle. I'm hoping this doesn't limit my progress to the RNG gods, but I suppose I'll find out. 

# What are you hoping for?
I'm hoping for something more than a visual novel and some different sorts of interactivity with the characters in the world. 

My guess is that it's a "room based" navigation system, but I'm curious about the adventure and RPG elements in it, if any.

With respect to inspiration, I am hoping to find a new or different to engage with a text driven game. 

# TL;DR; / Conclusion
My expectations aren't too high for this game, but I am hoping to find inspiration in the way a text-driven game engages with players. 

Stream starts tonight, July 20th at 9pm (UTC-5), on [Twitch](https://twitch.tv/davidwesst).

This is the first one, so here's hoping the tech forces of the universe are with me. 

Thanks for playing (and watching).

**Comment by davidwesst** [comment]
![20210719_161017.jpg](https://user-images.githubusercontent.com/867084/126227907-163b1682-85ee-47dc-8fce-460007c479a1.jpg)

**Comment by davidwesst** [comment]
Resolved with #22
